python/feast_spark/job_service.py

```diff
@@ -326,6 +362,12 @@ def ensure_stream_ingestion_jobs(client: Client, all_projects: bool):
         if all_projects
         else [client.feature_store.project]
     )
+    filtered_projects = []
+    if client._job_service._whitelisted_projects:
```

Comment:
I think you mixed up client of job service and server of job service.
Server has `_whitelisted_projects` property. Client doesn't.

Reply:
We may just use configuration here instead.